central/declarativeconfig/updater/auth_provider_updater.go

```diff
@@ -0,0 +1,44 @@
+package updater
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	authProviderDatastore "github.com/stackrox/rox/central/authprovider/datastore"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/authproviders"
+	"github.com/stackrox/rox/pkg/errox"
+)
+
+// AuthProviderUpdater handles updates of storage.AuthProvider within declarative config reconciliation routine.
+type AuthProviderUpdater struct {
```

Comment:
no reason to export those, right?
```suggestion
type authProviderUpdater struct {
```